_health-care/_js/_components/gender-input.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import _ from 'lodash';
+
+class GenderInput extends React.Component {
+  constructor() {
+    super();
+    this.state = { hasError: false };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentWillMount() {
+    this.id = _.uniqueId();
+  }
+
+  handleChange() {
+    const gender = this.refs.gender.value;
+
+    if (this.validate(gender)) {
+      this.setState({ hasError: false });
+    } else {
+      this.setState({ hasError: true });
+    }
+
+    this.props.onUserInput(gender);
+  }
+
+  validate(field) {
+    if (field === '') {
+      return true;
+    }
+    return /^[a-zA-Z '\-]+$/.test(field);
```

Comment:
I don't think you need this regex check because the field is a dropdown, so we're restricting what the user can input anyway